composer.json

```diff
@@ -0,0 +1,92 @@
+{
+    "name": "ttn/typo3-distribution",
+    "description": "Example distribution by the TYPO3 Trainer Network",
+    "type": "project",
+    "keywords": [
+        "typo3",
+        "example"
+    ],
+    "license": "GPL-2.0-or-later",
+    "authors": [
+        {
+            "name": "TYPO3 Trainer Network",
+            "email": "trainer-network@typo3.org"
+        }
+    ],
+    "support": {
+        "issues": "https://github.com/typo3-trainer-network/typo3-distribution/issues",
+        "source": "https://github.com/typo3-trainer-network/typo3-distribution"
+    },
+    "repositories": [
+        {
+            "type": "path",
+            "url": "localPackages/*"
+        }
+    ],
+    "require": {
+        "php": "~7.2 || ~7.3 || ~7.4",
+        "helmich/typo3-typoscript-lint": "^2.0",
+        "squizlabs/php_codesniffer": "^3.5",
+        "ttn/books": "^1.0",
+        "ttn/site-developer": "*",
+        "ttn/tea": "^2.0",
+        "typo3/cms-backend": "*",
+        "typo3/cms-core": "^9.5",
+        "typo3/cms-frontend": "*",
+        "typo3/cms-install": "*",
+        "typo3/cms-lowlevel": "*"
```

Comment:
For the CMS-related packaged, please let's always use a real version like `^9.5`, not the death star.

Reply:
We always use the version for `typo3/cms-core` and `*` for everything else. As they are only compatible with the exact same version, we only need to change one thing.

But I've adjusted it here.